Assets/CSS/style.css

```diff
@@ -0,0 +1,134 @@
+html, body {
+    height: 100%;
+  }
+  
+ /* Ensure background(s) fill whiteSpace */
+body,
+img,
+(contentDiv) {
+    background-attachment: fixed;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+  
+/* Apply googleFonts? */
+body {
+    background-image: url(#);
+    font-family: #;
+    font-size: 16px;
+    line-height: 1;
+}
+  
+/* Apply googleFonts? */
+h1 {
+    font-family: #;
+    font-weight: bolder;
+}
+  
+/* Add border-radius to divs for rounded corners? */
+(divCorners) {
```

Comment:
I don't think round brackets are valid CSS?

Reply:
I know they're not, they're placeholders ~ <_<